qiskit/circuit/quantumcircuit.py

```diff
@@ -4178,3 +4213,19 @@ def _insert_composite_gate_definition_qasm(
 
     string_temp = string_temp.replace(extension_lib, f"{extension_lib}{gate_definition_string}")
     return string_temp
+
+
+def _warn_on_broadcasting_controlled_gate(qargs, name):
+    # check for multiple qubits
+    if any(isinstance(qarg, (tuple, list, np.ndarray)) and len(qarg) > 1 for qarg in qargs):
+        warnings.warn(
+            "Creating multiple single-controlled gates with a single call, "
+            "as QuantumCircuit.cx([0, 1], 2) is deprecated as of Qiskit "
```

Comment:
```suggestion
            f"as QuantumCircuit.{name}([0, 1], 2) is deprecated as of Qiskit "
```

Reply:
I had that before but then discarded it for the `cx` example since for gates with more than two arguments (e.g. `ccx` or `cswap`) the syntax would not be right 😄 If you prefer we can also add a distinction for gates with 3 args but I thought the cx example is generic enough 🙂

Reply:
Oh no, good point - I hadn't thought of that.  Best leave it as you've done it.